src/components/modal-container.js

```diff
@@ -172,38 +169,51 @@ export default function ModalContainerFactory(
       this._closeModal();
     };
 
-    _onExportConfig = () => {
-      const {data} = this.props.uiState.exportData;
+    _onExportJSONMap = () => {
+      const {uiState} = this.props;
+      const {hasData} = uiState.exportMap[EXPORT_MAP_FORMAT.JSON];
 
       // we pass all props because we avoid to create new variables
-      const dump = data ? KeplerGlSchema.save(this.props)
```

Comment:
you have a problem with the variable name `dump`? :)

Reply:
i preferred to use data since now it is available :)